src/lib.rs

```diff
@@ -21,6 +20,8 @@
 use core::alloc::{Alloc, AllocErr, Layout};
 use core::cmp;
 use core::ptr;
+#[cfg(any(target_os = "linux", target_os = "macos", target_arch = "wasm32"))]
```

Comment:
Could the below default implementation of `Platform` be moved into a separate file like `dummy.rs`, and then could the `#[cfg]` get removed here?